### PR TITLE
Remove modlog ignore for voice verifcation

### PR DESCRIPTION
**Pull request by ChrisLovering**
When a user leaves and rejoins, and then asks for the voice verification role back in modmail, we want a way to see if they have every been given the role.

By removing this modlog ignore, it allows us to check the user-log channel for user updates to see if they were ever assigned the role.

### CODE REVIEW

**Comment by minalike** [comment]
As per the feedback from other mods, can you also remove the modlog ignore deleting `!voiceverify` messages?

**Review by MaskDuck — APPROVED** [review]
gud

**Review by minalike — CHANGES_REQUESTED** [review]
Changing my position to block this PR until we also remove the modlog ignore for invocations of !voiceverify 

https://github.com/python-discord/bot/blob/ee5a2166e08805a31dd6141db663ccacfde0b551/bot/exts/moderation/voice_gate.py#L242-L243

My justification is that when users contact the moderation team claiming they meet the requirements but still aren't able to pass voice gate, the first thing a moderator thinks to check is the message logs to see if they've correctly attempted the command. We can infer if they did based on relaying the DM logs with the bot, but those logs are in UTC and it's not as intuitive as simply searching the user ID in the message logs, nor is it guaranteed that a user had DMs open when attempting to verify. 

There's also no good reason I can foresee for ignoring these anyway. And who doesn't love removing code?

**Comment by ChrisLovering** [comment]
> Changing my position to block this PR until we also remove the modlog ignore for invocations of !voiceverify 
> 
> https://github.com/python-discord/bot/blob/ee5a2166e08805a31dd6141db663ccacfde0b551/bot/exts/moderation/voice_gate.py#L242-L243
> 
> My justification is that when users contact the moderation team claiming they meet the requirements but still aren't able to pass voice gate, the first thing a moderator thinks to check is the message logs to see if they've correctly attempted the command. We can infer if they did based on relaying the DM logs with the bot, but those logs are in UTC and it's not as intuitive as simply searching the user ID in the message logs, nor is it guaranteed that a user had DMs open when attempting to verify. 
> 
> There's also no good reason I can foresee for ignoring these anyway. And who doesn't love removing code? 

Yea I can agree with wanting to remove that,but let's not block this PR since it's still useful. I'm not able to make these code changes right now, so someone else can pickup the scope creep once this has been merged.

To that end I'll remove the issue link in the pr description so that it doesn't get closed.

**Review by minalike — APPROVED** [review]
So long as #2076 is not closed, then this is fine

**Review by D0rs4n — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
as per the gh actions you should remove event import

**Pull request merged**
